README.md

```diff
@@ -345,6 +345,21 @@ Test skeletons are codes that needs to be checked into the code base to describe
 
 ## Frontend test skeletons
 
+Frontend tests are done in [Cypress](https://cypress.io). Code is located at [`cypress`](cypress/). The test cases are in [`cypress/testcases`](cypress/testcases/). The way that they are organized are in `describe` and `it` blocks. Cypress uses [Mocha](https://mochajs.org/) to do [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development). An `it` block is a test case, and a `describe` block it a suite of tests with `it` blocks inside of it. 
+
+The test skeleton is just a stub method with JSDoc comments that add steps. Here's an example of a test skeleton for logging in. You would put this in the appropriate test spec in [`cypress/testcases`](cypress/testcases/). For login it would be [`login.spec.ts`](cypress/testcases/login.spec.ts). Since it's not implemented you will also add the `notImplemented` tag. 
+```
+/**
+* 1. Load login page
+* 2. Enter in username and password
+* 3. Click Login
+* 4. Verify that the dashboard loads
+* @notImplemented
+*/
+export function loginTest() {}
```

Comment:
Does it make more sense to include the `describe` and `it` part following? we can make it clear the skeleton doesn't include them but how it can look like in the end.

Reply:
I updated it with more explicit examples.